zebra/zebra_vty.c

```diff
@@ -91,6 +91,9 @@ struct static_hold_route {
 	char *tag_str;
 	char *distance_str;
 	char *label_str;
+
+	/* processed & masked destination, used for config display */
```

Comment:
why are you adding one line between prefix dest attribute and the other ones ?

Reply:
To distinguish it from the other fields? Why not?